hrjobcontract/CRM/Hrjobcontract/Export/Converter.php

```diff
@@ -43,6 +60,18 @@ public static function writeCSVFromTable($exportTempTable, $headerRows, $sqlColu
             list(, $entity, $field) = explode('_', $column, 3);
             $row[$column] = $converter->export($entity, $field, $row[$column]);
           }
+
+          if ($column === 'hrjobcontract_leave_leave_amount') {
+            $contractEntitlementsArray = [];
+            $contractEntitlements = explode(',', $row[$column]);
+            foreach ($contractEntitlements as $contractEntitlement) {
+              list($leaveType, $entitlement) = explode(':', $contractEntitlement);
+              $contractEntitlementsArray[trim($leaveType)] = trim($entitlement);
+            }
+            foreach ($leaveTypes as $type => $typeId) {
+              $row[$type] = !empty($contractEntitlementsArray[$type]) ? $contractEntitlementsArray[$type] : '';
+            }
```

Comment:
If I got this right, the exported file will always have columns for all the absence types, because some contacts might have entitlement for an absence type and others don't. Is this correct? If so, I think it would be nice to add a comment here explaining this